overlord/devicestate/devicestate.go

```diff
@@ -1179,7 +1202,7 @@ func checkForInvalidSnapsInModel(model *asserts.Model, vSets *snapasserts.Valida
 //     (need to check that even unchanged snaps are accessible)
 //   - Make sure this works with Core 20 as well, in the Core 20 case
 //     we must enforce the default-channels from the model as well
-func Remodel(st *state.State, new *asserts.Model, localSnaps []*snap.SideInfo, paths []string) (*state.Change, error) {
+func Remodel(st *state.State, new *asserts.Model, offline bool, localSnaps []*snap.SideInfo, paths []string) (*state.Change, error) {
```

Comment:
we should use a RemodeOptions struct even if there is only one field for now, opts *RemodelOptions would come last in the args

Reply:
Should the local snap slices also go in there? Since they are optional?

Reply:
I would say no for now